### ISSUE TITLE
Question: How can I download a Youtube video, and just the video and English subtitles?

### ISSUE THREAD

**Issue by aidenmitchell** [ArchiveBox/ArchiveBox#890]
Apologies for another issue, I don't mean to spam.

Currently I have `SAVE_MEDIA = False` in my config, but I'm getting lots of `.vtt` files downloaded, and the archive size gets big.
Is there a way to archive only:
1. the original video, in 1080p
2. english subtitles
3. the `.description` file

Thanks for your help!

**Comment by pirate** [comment]
You can add arbitrary YoutubeDL arguments using `archivebox config --set YOUTUBEDL_ARGS="..."`, just figure out what you need and modify the existing args from `archivebox/config.py:YOUTUBEDL_ARGS` and pass them into that config setting.

**Comment by aidenmitchell** [comment]
Thanks, but it's not too happy about that unfortunately.
```logs
archivebox config --set YOUTUBEDL_ARGS="--write-sub"
[i] [2021-11-13 01:37:10] ArchiveBox v0.6.3: archivebox config --set YOUTUBEDL_ARGS=--write-sub
    > /data


[X] Error while loading configuration value: YOUTUBEDL_ARGS
    JSONDecodeError: Expecting value: line 1 column 1 (char 0)

    Check your config for mistakes and try again (your archive data is unaffected).

    For config documentation and examples see:
        https://github.com/ArchiveBox/ArchiveBox/wiki/Configuration
```
I'm probably doing something wrong, however I also copy-pasted your example command and it threw the same error.

**Comment by remyabel** [comment]
It's a JSON list, not a string.

**Comment by remyabel** [comment]
@aidenmitchell Try this instead:

```
archivebox config --set YOUTUBEDL_ARGS='["--write-description", "--write-info-json", "--write-annotations", "--write-thumbnail", "--no-call-home", "--write-sub", "--all-subs", "--write-auto-sub", "--convert-subs=srt", "--yes-playlist", "--continue", "--ignore-errors", "--geo-bypass", "--add-metadata", "--max-filesize=500m", "--sub-lang=en"]'
```

**Comment by pirate** [comment]
Right, sorry I forgot it needs to be a JSON list!

**Comment by aidenmitchell** [comment]
@remyabel Thank you! Works now.

**Comment by dohlin** [comment]
> @aidenmitchell Try this instead:
> 
> ```
> archivebox config --set YOUTUBEDL_ARGS='["--write-description", "--write-info-json", "--write-annotations", "--write-thumbnail", "--no-call-home", "--write-sub", "--all-subs", "--write-auto-sub", "--convert-subs=srt", "--yes-playlist", "--continue", "--ignore-errors", "--geo-bypass", "--add-metadata", "--max-filesize=500m", "--sub-lang=en"]'
> ```

I'm running this but still hitting the following error:

`docker-compose run archivebox config --set YOUTUBEDL_ARGS='["--write-description", "--write-info-json", "--write-annotations", "--write-thumbnail", "--no-call-home", "--write-sub", "--all-subs", "--write-auto-sub", "--convert-subs=srt", "--yes-playlist", "--continue", "--ignore-errors", "--geo-bypass", "--add-metadata", "--max-filesize=500m", "--sub-lang=en"]'
Creating archivebox_archivebox_run ... done
[i] [2023-02-28 15:18:17] ArchiveBox v0.6.2: archivebox config --set YOUTUBEDL_ARGS=[--write-description, --write-info-json, --write-annotations, --write-thumbnail, --no-call-home, --write-sub, --all-subs, --write-auto-sub, --convert-subs=srt, --yes-playlist, --continue, --ignore-errors, --geo-bypass, --add-metadata, --max-filesize=500m, --sub-lang=en]
    > /data

usage: archivebox config [-h] [--get | --set | --reset] [config_options ...]
archivebox config: error: unrecognized arguments: --write-info-json, --write-annotations, --write-thumbnail, --no-call-home, --write-sub, --all-subs, --write-auto-sub, --convert-subs=srt, --yes-playlist, --continue, --ignore-errors, --geo-bypass, --add-metadata, --max-filesize=500m, --sub-lang=en]
ERROR: 2
`

**Comment by dohlin** [comment]
Ok that didn't work either unfortunately as it just led to the 'continued input' line in Ubuntu (I'm probably using the wrong term there, but basically what happens when you end a command with a slash.

However, based on your info about docker stripping off the singlequotes I figured out the command that works in my case.  Here it is in case anyone else stumbles upon this later:

docker-compose run archivebox config --set YOUTUBEDL_ARGS=\''["--write-description", "--write-info-json", "--write-annotations", "--write-thumbnail", "--no-call-home", "--write-sub", "--all-subs", "--write-auto-sub", "--convert-subs=srt", "--yes-playlist", "--continue", "--ignore-errors", "--geo-bypass", "--add-metadata", "--max-filesize=500m", "--sub-lang=en"]'\'

Basically just escaping the _outermost_ singlequotes.  Thank you!
